app/src/androidTest/java/ch/sdp/vibester/fragment/GameSetupFragmentTest.kt

```diff
@@ -26,45 +33,58 @@ import org.junit.runner.RunWith
 
 @RunWith(AndroidJUnit4::class)
 @HiltAndroidTest
-class GameSetupActivityTest {
+class GameSetupFragmentTest {
 
     @get:Rule(order = 0)
     var hiltRule = HiltAndroidRule(this)
 
-    @get: Rule(order = 1)
-    val activityRule = ActivityScenarioRule(GameSetupActivity::class.java)
+    @After
+    fun clean() {
+        Intents.release()
+    }
 
     @Before
-    fun setUp() {
+    fun setup() {
         hiltRule.inject()
         Intents.init()
+        launchFragmentInHiltContainer<GameSetupFragment>(
+            themeResId = R.style.AppTheme
+        )
     }
 
-    @After
```

Comment:
We don't need this function anymore?

Reply:
I think we do. what happens to the intent fired during the tests?

Reply:
okk,  I thought you removed it